src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css

```diff
@@ -0,0 +1,151 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-gray-300);
+  grid-column: 1 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 2;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: auto 590px;
+  padding-top: 122px;
+  row-gap: 84px;
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 80px;
+    padding-top: 50px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaTopLeft {
+  grid-column: 2 / span 4;
+  grid-row: 1;
+}
+
+.gridAreaTopRight {
+  grid-column: 6 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaBottomLeft {
+  grid-column: 2 / span 5;
+  grid-row: 2;
+}
+
+.gridAreaBottomRight {
+  grid-column: 8 / span 4;
+  grid-row: 2;
+}
+
+
+/* Component wrappers */
+
+.paragraph1Wrapper {
+  font: var(--font-body-large);
+  margin-top: 0;
+}
+
+.paragraph2Wrapper {
+  font: var(--font-body-small);
+  margin-top: 0;
+}
+
+.paragraph2Wrapper p:first-child {
+  margin-top: 0;
+}
+
+.paragraph2Wrapper p:last-child {
+  margin-bottom: 0;
+}
+
+.image1Wrapper {
+  position: relative;
+  top: 35px;
+}
+
+@media (--tablet-and-down) {
+  .paragraph1Wrapper {
+    margin-right: 40px;
+    margin-top: 40px;
+  }
+
+  .paragraph2Wrapper {
+    margin-right: 40px;
+    margin-top: 40px;
+  }
```

Comment:
You actually need to override the font sizes on mobile. This is especially true of `.paragraph2Wrapper`, where the font size is way too small for significant body text on a mobile device.

Reply:
Thanks for noticing this! I made the change for `.paragraph1Wrapper` mobile font size to 16px. However, I noticed on the Figma design that the mobile font size for `.paragraph2Wrapper` is 16px which is what it's currently set to.

Reply:
You're right that the font size should be 16px, but when I check out and run this branch, I see that it's getting a size of 14px.

I think there's a slight inconsistency in the mobile Figma design, where the Typography section in the right side bar says that the name of the style is "Web / Small Font", even though it's a mobile design, and therefore should use the Mobile font styles. "Web / Small Font" is 16px, but "Mobile / Small Font" is 14px, and instead "Mobile / Medium Font" is actually 16px.